TheSufferfest/node-fit
# WB-70 Include lap object in sessions

#### What’s this PR do?
Adds the lap object to the array "laps" in sessions 
#### Any background context you want to provide?
This update adds a lap object with the following struture 
```
{
 startTime:1,
 totalElapsedTime:13749,
 totalDistance:5.73,
 maxSpeed:0.368,
 avgSpeed:0.417
}
```
#### How should this be manually tested?
`npm install`
`npm test`
It should pass the test
Go to examples\content-to-js
`npm install`
`node index.js`
Verify that a file called temp.fit
Open the fit file in a desktop application like Golden Cheetah and review the content. (This file is created from NodeJS)
#### Could any automated tests be added for this?
Maybe
#### What are the relevant JIRA issues?
https://thesufferfest.atlassian.net/browse/WB-70
#### Screenshots (if appropriate)
#### Open issues
